tests/marine/benchmark/benchmark_generator.py

```diff
@@ -0,0 +1,68 @@
+from ipaddress import IPv4Address
+from random import randint, getrandbits
+from typing import List
+
+from .conversation_generators import (
+    generate_raw_tcp_conversation,
+    generate_raw_udp_conversation,
+)
+from .utils import ConversationGenerator, Layer3Conversation, BenchmarkPacket
+
+CONVERSATION_GENERATORS = [
+    ConversationGenerator(0.5, generate_raw_tcp_conversation),
+    ConversationGenerator(0.5, generate_raw_udp_conversation),
+]
+
+
+def create_random_mac() -> str:
+    return "00:00:00:%02x:%02x:%02x" % (
+        randint(0, 255),
+        randint(0, 255),
+        randint(0, 255),
+    )
+
+
+def generate_macs(count: int) -> List[str]:
+    if count % 2 != 0:
+        count += 1
+    macs = set()
+    while len(macs) < count:
+        macs.add(str(create_random_mac()))
+    return list(macs)
+
+
+def generate_ips(count: int) -> List[str]:
+    if count % 2 != 0:
+        count += 1
+    ips = set()
+    while len(ips) < count:
+        ips.add(str(IPv4Address(getrandbits(32))))
+    return list(ips)
```

Comment:
Pretty sure that will return the ips sorted by their hash values (which I'm pretty sure translates to "sorted lexicographically")
It's not as random you'd expect, but I don't think it really matters.

Reply:
Good point! I'll shuffle the list